src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const RENT_COST = 40;
+
+  const RESULT = days * RENT_COST;
```

Comment:
Consider renaming 'RESULT' to a more descriptive name such as 'totalCost' to clearly reflect that it represents the cost before any discounts are applied.